packages/libs/client-utils/src/nodejs/marmalade/deployment/defaults.ts

```diff
@@ -0,0 +1,91 @@
+import { createSignWithKeypair } from '@kadena/client';
+import type { ChainId } from '@kadena/types';
+import type { IAccount, IClientConfig } from '../../../core/utils/helpers';
+import type {
+  ILocalConfig,
+  INamespaceConfig,
+  IRemoteConfig,
+  IRepositoryConfig,
+} from './config';
+
+export interface IAccountWithSecretKey extends IAccount {
+  secretKey: string;
+}
+
+export const defaultAccount: IAccountWithSecretKey = {
+  account: 'sender00',
+  publicKeys: [
+    '368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca',
+  ],
+  secretKey: '251a920c403ae8c8f65f59142316af3c82b631fba46ddea92ee8c95035bd2898',
+};
+
+export const defaultChainId: ChainId = '0';
+export const defaultNetworkId = 'fast-development';
+export const defaultNetworkHost = 'http://localhost:8080';
+
+/* Defines the arguments for the marmalade contracts: these values are going
+to be replaced in the marmalade templates and code files  */
```

Comment:
We can use actual TSdoc here. 

```suggestion
/** 
 * Defines the arguments for the marmalade contracts: these
 * values are going to be replaced in the marmalade 
 * templates and code files  
 */
```

_Idem for the other occurrences_